src/main/target/KAKUTEF4/target.h

```diff
@@ -126,7 +126,15 @@
 #define RSSI_ADC_CHANNEL            ADC_Channel_11
 
 #define LED_STRIP
-#define LED_STRIP_TIMER         TIM5
+#define WS2811_PIN                      PA0
```

Comment:
I add "DEF_TIM(TIM5, CH1, PA0, TIM_USE_LED,   TIMER_OUTPUT_ENABLED, 0),  // LED_STRIP - DMA1_ST2" in target.c and rebuild the firmware, but i found that led_trip still can not work.
After I add PIN definitions in target.h,  led_trip  work  well.